src/Features/Core/Portable/Completion/CompletionHelper.cs

```diff
@@ -358,5 +359,109 @@ private static int CompareExpandedItem(CompletionItem item1, PatternMatch match1
 
         public static string ConcatNamespace(string? containingNamespace, string name)
             => string.IsNullOrEmpty(containingNamespace) ? name : containingNamespace + "." + name;
+
+        internal static bool TryCreateMatchResult<T>(
```

Comment:
as above, this is moved code, the main change being the removal of the call to get highlighted spans